<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSOSSourceSelectBase</class>
 <widget class="QDialog" name="QgsSOSSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add SOS Layer from a Server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="GroupBox1">
     <property name="title">
      <string>Server connections</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="mConnectionsComboBox"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="mConnectButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>C&amp;onnect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mNewButton">
          <property name="text">
           <string>&amp;New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bmEditButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mDeleteButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>171</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="mLoadButton">
          <property name="toolTip">
           <string>Load connections from file</string>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mSaveButton">
          <property name="toolTip">
           <string>Save connections to file</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="mbuttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeView" name="mOfferingsTreeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mConnectionsComboBox</tabstop>
  <tabstop>mConnectButton</tabstop>
  <tabstop>mNewButton</tabstop>
  <tabstop>bmEditButton</tabstop>
  <tabstop>mDeleteButton</tabstop>
  <tabstop>mOfferingsTreeView</tabstop>
  <tabstop>mbuttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
